ocean_lib/config.py

```diff
@@ -273,7 +273,7 @@ def network_name(self) -> str:
     @property
     @enforce_types
     def chain_id(self) -> int:
-        """Chain ID of the ethereum network. (e.g.): 1337."""
+        """Chain ID of the ethereum network. (e.g.): 8996."""
```

Comment:
I recommend changing to "Chain ID of the EVM-compatible network." and removing the example.

Reply:
Thank you.